src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -713,6 +713,7 @@ def load_arguments(self, _):
         c.argument('reset', help='Reset Java apps to the default parking page if set to true with no type specified.', arg_type=get_three_state_flag())
         c.argument('timeout', type=int, help='Timeout for the deployment operation in milliseconds. Ignored when using "--src-url" since synchronous deployments are not yet supported when using "--src-url"')
         c.argument('slot', help="The name of the slot. Default to the productions slot if not specified.")
+        c.argument('track_deployment', help="If true, web app deployment status will be tracked verbosely.", arg_type=get_three_state_flag())
```

Comment:
Maybe should we call this `track_runtime_status` or `poll_new_status` or something better?
let's get the review from @tulikac and @btardif  on the new param name and the general approach.

Reply:
Hi @sarsharma , is there any update on the param name? 

Please note that we are launching the release for this sprint this week. Please resolve all comments by 11/07/2023 10:00 UTC, otherwise the release of this PR will have to be postponed to the next sprint **(on 12-05)**.

Reply:
switching the param name to `track-runtime-status`